### ISSUE TITLE
Confusing concept of "self-supervised learning"

### ISSUE THREAD

**Issue by fqnchina** [JannerM/intrinsics-network#11]
This paper is named self supervised intrinsic image decomposition. However, it means self-supervised transfer from a fully-supervised pre-trained model. This is kind of misleading for me, as I haven't be sure about this until I check the codes.

**Comment by jannerm** [comment]
Thanks for bringing this up. We call our approach 'self-supervised learning' because it learns in a self-supervised manner on novel object classes (bunny, teapot, etc.) after pre-training on simple geometric primitives. No annotations are required for the new classes. Here is our description of the setup from the paper:

> In the following experiments, we first train RIN (including the shading model) on a dataset with ground truth labels for intrinsic images. This is treated as a standard **supervised learning problem** using mean squared error on the intrinsic image predictions as a loss. The model is then trained further on an additional set of unlabeled data using only reconstruction loss as an error signal. We refer to this as the **self-supervised transfer**.

**Comment by fqnchina** [comment]
Got it. Thanks for the clarification.
